py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -26,9 +37,10 @@
     get_node_name,
     get_trt_tensor,
 )
-from torch_tensorrt.dynamo.utils import DYNAMIC_DIM
+from torch_tensorrt.dynamo.utils import DYNAMIC_DIM, to_torch_device
 from torch_tensorrt.fx.observer import Observer
 from torch_tensorrt.logging import TRT_LOGGER
+from tqdm import tqdm
```

Comment:
We already have an optional dependency on rich coming in another PR. We should enable this optionally

Reply:
Ok I will delete this for now and we can add that later